website/client/src/components/shops/buyModal.vue

```diff
@@ -119,7 +122,10 @@
         <div v-if="attemptingToPurchaseMoreGemsThanAreLeft">
           {{ $t('notEnoughGemsToBuy') }}
         </div>
-        <div v-if="nonSubscriberHourglasses" class="hourglass-nonsub mt-3">
+        <div
```

Comment:
@tsukimi2 looks like your text editor might be autoformatting the code in some places, is it possible to disable that behavior? Because it ends up inserting a lot of unrelated changes inside PRs that make it a bit difficult to see what actually changed

Reply:
@paglias Yes, sorry, I noticed this problem before also, and I thought it was due to lint modifying the files after running "npm run lint" (since when I run git status after making code changes but before running npm run lint, the git status showed only the files I made changes as being modified, but after running npm run lint, a bunch of files that I have not made any changes myself got marked as modified).  I always thought it's lint modifying the files after running "npm run lint" and not noticing it may be due to the visual studio code that I'm using (first time using this ide on a project) to edit code.  I've just set its editor.formatOnSave to false in its settings.json now and see whether this fixes the problem next time I submit a PR, or I might simply try using a barebone vi to edit code first and see what happens.  Sorry for the inconvenience this has caused.

Reply:
No worries, it's happening quite often with other PRs as well. I thought about `npm run lint` as well but I don't think it's, VSCode is more likely but I'm not sure if it's just applying some extra lint rules (warns instead of errors) that aren't applied by `npm run lint` in which case it's totally fine or if it's using it's own rules. I'll let you know if I find what's causing this